internal/backend/local/local_unix.go

```diff
@@ -9,6 +9,6 @@ import (
 )
 
 // set file to readonly
-func setNewFileMode(f string, mode os.FileMode) error {
-	return fs.Chmod(f, mode)
+func setFileReadonly(f string, mode os.FileMode) error {
+	return fs.Chmod(f, mode&os.FileMode(^uint32(0222)))
```

Comment:
More modern Go would be

     mode &^ 0222

Reply:
I've changed it, That's indeed much more readable.